src/minidriver/cardmod-mingw-compat.h

```diff
@@ -0,0 +1,28 @@
+#define __deref
```

Comment:
@alonbl What do you think? Would you agree to add your file to OpenSC under OpenSC's license?